### ISSUE TITLE
MatrixtoSimplicialComplexTriangular - where?

### ISSUE THREAD

**Issue by muschellij2** [lcrawlab/SINATRA#2]
Not sure where MatrixtoSimplicialComplexTriangular is defined, but it's not in there.

**Comment by muschellij2** [comment]
Similarly - `check_reconstruction`

**Comment by brwaang55** [comment]
Good catch. The `MatrixtoSimplicialComplexTriangular` function is used for an older round of simulations where we used rbf-kernel interpolations, the current setting for `generate_roc_curve_x` is to specify `mode='sphere'`. The functions will be updated accordingly. 

I'd have to get back to you on `check_reconstruction` @tsudijon  can you help out here?

**Comment by tsudijon** [comment]
Thanks for raising this issue -- the `check_reconstruction` function is used in simulations which assess SINATRA's performance (using an ROC-type metric) and its sensitivity to parameters, in particular the functions `roc_curve_simulation` and `metric_curve_simulation`. The `check_reconstruction` function was used for an older use-case of these simulation functions, where the `type` parameter is given by `cone`. In all simulations reported, we use `type=vertex` which does not use the `check_reconstruction` function.

The `type` parameter controls how quality of the reconstruction is measured; we settled on `type = vertex`, which compares vertex-by-vertex the constructed region on a given shape to the known causal region on the same shape. The `type=cone` metric was something we experimented with earlier but was not as intuitive as a vertex-by-vertex comparison.

Potentially makes sense to remove the `type='cone'` functionality in these two simulation functions.
